ios/FluentUI/Tab Bar/TabBarTokens.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Design token set for the `TabBar`.
+open class TabBarTokens: ControlTokens {
+    /// The height of the `TabBar` when in portrait view on a phone
+    open var phonePortraitHeight: CGFloat { 48.0 }
+
+    /// The height of the `TabBar` when in landscape view on a phone
+    open var phoneLandscapeHeight: CGFloat { 40.0 }
+
+    /// The height of the `TabBar` when on a non-phone device
+    open var padHeight: CGFloat { 48.0 }
+
+    /// Defines the background color of the  of the `TabBarItem` when selected.
+    open var selectedColor: DynamicColor {
+        return self.aliasTokens.foregroundColors[.brandRest]
+    }
+
+    /// Defines the background color of the  of the `TabBarItem` when not selected.
+    open var unselectedColor: DynamicColor {
+        DynamicColor(light: ColorValue(0x6E6E6E) /* gray500 */,
+                            lightHighContrast: ColorValue(0x303030) /* gray700 */,
+                            dark: ColorValue(0x919191) /* gray400 */,
+                            darkHighContrast: ColorValue(0xC8C8C8) /* gray200 */)
```

Comment:
Looks like the spacing on the values other than light is a bit odd. Hitting control + i should align them.